### DISCUSSION TITLE
Is your presentation from mlops.community available online?

### DISCUSSION THREAD

**Discussion by elbamos** [fastapi/fastapi#9434] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# No code - I'm asking for a link to a presentation
```

### Description

You gave a presentation at mlops.community two weeks ago. Is the presentation available online? Thanks!

### Operating System

Linux

### FastAPI Version

0.95.1

### Python Version

3.10.10

**Comment by tiangolo** [comment, marked as answer]
Hello! I think they recorded it and will make it available online at some point, but I don't really know. They will most probably announce it in their social networks when that becomes available. 😁

**Reply by elbamos** [reply]
Thanks, but I'm asking about the slides you presented?  You had some design patterns in the slides I wanted to use.
